pkg/client/proxy_backup.go

```diff
@@ -108,18 +104,16 @@ func (c *ProxyClient) BackupRestore(serviceAddress, url, target, volumeName stri
 	}
 	recv, err := c.service.BackupRestore(c.ctx, req)
 	if err != nil {
-		return err
+		return errors.Wrapf(err, "%v failed to restore backup %v to volume %v", c.getProxyErrorPrefix(serviceAddress), url, volumeName)
```

Comment:
Should we just keep the original defer function to make sure we have a consistent error format? Also found other methods in this file having the same issues?

```golang
	defer func() {
	    if err != nil {
		    err = errors.Wrapf(err, "%v failed to restore backup %v to volume %v", c.getProxyErrorPrefix(serviceAddress), url, volumeName)
		}
	}()
```

Reply:
Yes. Addressed!